pkg/install/v1alpha1/jiva_volume_0.7.0.go

```diff
@@ -428,6 +428,7 @@ spec:
     {{- jsonpath .JsonResult "{.items[*].status.podIP}" | trim | saveAs "readlistrep.podIP" .TaskResult | noop -}}
     {{- jsonpath .JsonResult "{.items[*].status.containerStatuses[*].ready}" | trim | saveAs "readlistrep.status" .TaskResult | noop -}}
     {{- jsonpath .JsonResult "{.items[*].metadata.annotations.openebs\\.io/capacity}" | trim | saveAs "readlistrep.capacity" .TaskResult | noop -}}
+    {{- jsonpath .JsonResult "{.items[*].spec.volumes[0].hostPath.path}" | trim | saveAs "readlistrep.hostPath" .TaskResult | noop -}}
```

Comment:
A similar change may be required for 0.6.0 CAS template. Also, in case this value is not present - does the mayactl display - black or unknown?